recipes/iso3166/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "iso3166" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b1e58dbcf50fbb2c9c418ec7a6057f0cdb30b8f822ac852f72e71ba769dae8c5
+
+build:
```

Comment:
This looks like it could be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python